api/routes/connection.ts

```diff
@@ -92,6 +96,7 @@ export default async function router(schema: Schema, config: Config) {
             description: Default.DescriptionField,
             enabled: Type.Optional(Type.Boolean({ default: true })),
             agency: Type.Union([Type.Null(), Type.Optional(Type.Integer({ minimum: 1 }))]),
+            integrationId: Type.Union([Type.Integer(), Type.Null()]),
```

Comment:
Can we update this to use `Type.Optional` instead of `null`? In the deployments outside of COTAK/WFTAK the user shouldn't have to know/care about `integrationId` as it won't exist as a concept while agency can/will be mocked from the TAK server instead of the COTAK api in these cases.